tests/test_owner_privilages.py

```diff
@@ -1,14 +1,47 @@
+from flask import g
+
+from datetime import datetime
+
 from timeless.access_control.methods import Method
 from timeless.access_control.owner_privileges import has_privilege
+from timeless.employees.models import Employee
 
 
 def test_can_access_location():
     assert (
-        has_privilege(method=Method.CREATE, resource="location") == True
+            has_privilege(method=Method.CREATE, resource="location") is True
     )
 
+
 def test_cant_access_unknown_resource():
     assert (
-        has_privilege(method=Method.CREATE, resource="unknown") == False
+            has_privilege(method=Method.CREATE, resource="unknown") is False
+    )
+
+
+def test_cant_access_his_profile():
+    assert (
+            has_privilege(method=Method.READ, resource="employee", employee_id=1) is False
     )
 
+
+def test_can_access_his_profile():
+    g.user = Employee(id=1, first_name="Alice", last_name="Cooper",
```

Comment:
@Vatavuk Actually I'm not sure it's a good thing to set `g.user` here without any other "magic", since I think `pytest` won't clean `g` object between tests. Need to check it, mb it's okm otherwise it will affect all tests in the project

Reply:
@vryazanov I will clean it manually

Reply:
@vryazanov is it ok if I leave puzzle for this? It looks like we need some magic sauce:D

Reply:
@Vatavuk yes :+1: , I guess we need to write pytest fixture for this.